api-report/telemetry-utils.api.md

```diff
@@ -207,9 +207,6 @@ export class MultiSinkLogger extends TelemetryLogger {
 // @public
 export function normalizeError(error: unknown, annotations?: IFluidErrorAnnotations): IFluidErrorBase;
 
-// @public
-export function originatedAsExternalError(e: any): boolean;
```

Comment:
should this be removed in main? or should it be deprecated? and removed in next? it looks publicly exported and doesn't have an internal ts-doc

Reply:
It's internal. Lots of stuff in this package is, I opened this task to add that markup:  AB#712